src/app/pages/dashboard/widgets/all-widgets.constant.ts

```diff
@@ -38,49 +38,6 @@ import { systemInfoPassiveWidget } from 'app/pages/dashboard/widgets/system/widg
 import { systemImageWidget } from 'app/pages/dashboard/widgets/system/widget-system-image/widget-system-image.definition';
 import { systemUptimeWidget } from 'app/pages/dashboard/widgets/system/widget-system-uptime/widget-system-uptime.definition';
 
-export const widgetComponents = [
```

Comment:
We have `README.md` (src/app/pages/dashboard/widgets/README.md) with 

5. Connect widget definition to `widgetComponents` and `widgetRegistry.` text